pkg/autoscaler/autoscaler.go

```diff
@@ -291,6 +296,138 @@ var _ = Describe("Autoscaler should", framework.LabelAutoscaler, Serial, func()
 			}, framework.WaitLong, pollingInterval).Should(BeTrue(), "MachineSet %s failed to scale in to 0 replicas", machineSet.GetName())
 		})
 
+		// Machines required for test: 1
```

Comment:
probably need to update this to reflect the variable nature of the test now

Reply:
done.